compiler/modules/types/core.bal

```diff
@@ -1031,21 +1002,16 @@ public function listAtomicSimpleArrayMemberType(ListAtomicType? atomic) returns
 final ListAtomicType LIST_ATOMIC_TOP = { members: { initial: [], fixedLength: 0 }, rest: TOP };
 
 final readonly & ListMemberTypes LIST_MEMBER_TYPES_ALL = [[{ min: 0, max: int:MAX_VALUE }], [TOP]];
-final readonly & ListMemberTypes LIST_MEMBER_TYPES_READONLY = [[{ min: 0, max: int:MAX_VALUE }], [READONLY]];
 final readonly & ListMemberTypes LIST_MEMBER_TYPES_NONE = [[], []];
 
 public function listAllMemberTypes(Context cx, SemType t) returns ListMemberTypes {
-    if t is UniformTypeBitSet {
-        if t == LIST_RO {
-            return LIST_MEMBER_TYPES_READONLY;
-        }
-        return (t & LIST_RW) != 0 ? LIST_MEMBER_TYPES_ALL : LIST_MEMBER_TYPES_NONE;
+    if t is BasicTypeBitSet {
+        return (t & LIST) != 0 ? LIST_MEMBER_TYPES_ALL : LIST_MEMBER_TYPES_NONE;
```

Comment:
Seem to have another test failure outside the compiler module. Please refer [d3b4805](https://github.com/ballerina-platform/nballerina/pull/1063/commits/d3b4805987880ee266c75ffaceff141625580e33) for the failing test. 
@jclark @heshanpadmasiri any guesses on this?